package.json

```diff
@@ -20,7 +20,7 @@
   },
   "bugs:": "https://github.com/dnsimple/dnsimple-node/issues",
   "engines": {
-    "node": ">= v6.0.0"
+    "node": ">= v12.0.0"
```

Comment:
⚠️ This would require a major version bump. It will also prevent any customer not using latest node to not be able to use the client. Is it needed?

Reply:
It was a test to see if we could build the latest. I have now Travis configured to run the 3 versions supported by node: support/maintenance/current. So I am going to down to v8.